gradle.properties

```diff
@@ -5,9 +5,9 @@ android.useAndroidX=true
 #enabledSandboxResourceProcessor=false
 enabledSandboxResourceProcessor=true
 enableKotlinNative=true
-enableKotlinNativeLinux=true
+enableKotlinNativeLinux=false
 enableKotlinMobile=true
-enableKotlinAndroid=true
+enableKotlinAndroid=false
```

Comment:
to revert later

Reply:
did you have any kind of errors because of this? Would be nice to figure out the root cause so other people can benefit from it

Reply:
Using :
![image](https://user-images.githubusercontent.com/919982/210138200-813683dd-d759-4f3b-a885-05982932fc41.png)

when enabling : 

`Could not find compile target android-30 for modules :kbignum, :kds, :klock, :klogger, :kmem, :korau, :korge, :korgw, :korim, :korinject, :korio, :korma, :korte, :korvi, :krypto, :luak, :graphics:foundation, :graphics:backend:metal`  On IntelliJ